src/aap_eda/tasks/orchestrator.py

```diff
@@ -191,86 +217,106 @@ def dispatch(
         ):
             if not queue_name:
                 LOGGER.info(
-                    "Scheduling request"
-                    f"{request_type_text} for {process_parent_type}"
-                    f" {process_parent_id} to the least busy queue"
-                    "; it is not currently associated with a queue.",
+                    "Scheduling request "
+                    f"{request_type} for {process_parent_type} "
+                    f"{process_parent_id} to the least busy queue; "
+                    "it is not currently associated with a queue.",
                 )
             else:
                 LOGGER.info(
                     "Scheduling request"
-                    f"{request_type_text} for {process_parent_type}"
-                    f" {process_parent_id} to the least busy queue"
-                    f"; its associated queue '{queue_name}' is from"
-                    " previous configuation settings.",
+                    f"{request_type} for {process_parent_type} "
+                    f"{process_parent_id} to the least busy queue; "
+                    f"its associated queue '{queue_name}' is from "
+                    "previous configuation settings.",
                 )
-            queue_name = None
-
-    if queue_name and (not check_rulebook_queue_health(queue_name)):
-        # The queue is unhealthy; log this fact.
-        # We'll try to find another queue on which to run.
-        msg = (
-            f"{process_parent_type} {process_parent_id} is in an"
-            " unknown state. The workers of its associated queue"
-            f" '{queue_name}' are failing liveness checks."
-            " There may be an issue with the node; please contact"
-            " the administrator."
-        )
-        LOGGER.warning(
-            "Request"
-            f"{request_type_text} for {process_parent_type}"
-            f" {process_parent_id} will be run on the least busy queue."
-        )
-        queue_name = None
+            try:
+                queue_name = get_least_busy_queue_name()
```

Comment:
This is the case when request type is not start or auto-start, which must be stop/restart/monitor. Now we cannot find its original queue. Can these operations be successfully proceeded by a new queue (node)?
In the below logic where queue is valid but not healthy, you don't proceed the such operations by another queue unless it is restart.

Reply:
This block is to cover the case where there is a change in the settings for the queues. For instance, where a node is removed or when we have a new deployment created from backup/restore (customers do that to promote a staging env to a production or to create a test env from production). In a regular scenario, the activation is always related to a queue (expected node), so we reschedule it only when it is explicitly restarted, because the node associated might still be running the activation, so we don't to duplicate it unless the user/admin makes sure it can be restarted.